dev/integration_tests/flavors/ios/Runner.xcodeproj/xcshareddata/xcschemes/free.xcscheme

```diff
@@ -27,15 +27,6 @@
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
       shouldUseLaunchSchemeArgsEnv = "YES">
-      <MacroExpansion>
```

Comment:
Xcode 12 didn't like this and removed it.

Reply:
rip